AGPs/The-Aragon-Network-is-not-Game-of-Thrones.md

```diff
@@ -0,0 +1,38 @@
+---
+## AGP: N/A
+Title: AN ≠ GoT
+Author: burrrata (@burrrata)
+Status: Stage III
+Track: Meta
+Created: 2019-11-19
+---
+
+# AGP-X: AN ≠ GoT
+
+## File changed (AGP-0 or AGP-1)
+
+AGP-1
+
+## Link to proposed change
+
+https://github.com/aragon/AGPs/pull/146
+
+## Motivation for making this change
+
+AGP-103 was marketed as a way to minimize spending by enforcing a budget. This is accomplished via a 2m cap on finance-track spending, however, this is not the only consequence of this proposal. 
+
+AGP-103 favors recurring subscriptions with the maximum amount of absolute support. It also favors current recurring proposals over new proposals. Let's explore what that really means...
+
+This incentivizes teams to ask for as much money as possible as early as possible. It's a SaaS model, but for Aragon treasury funding. It's easy to sign up for (ANT holders vote yes on almost everything), and hard to cancel (to date an AGP has never been reversed).
+
+AGP-103 favors current recurring payments over new proposals asking for funds. The teams who are currently in the Aragon ecosystem will benefit greatly from this. New teams will be discouraged to get involved in the network. If they do want to get involved they will have to find a way to make room in the budget for their projects. 
+
+To address the fact that AGPs have never been reversed new proposals cancelling currently recurring proposals will be favored before all others. Note: this does not mean that a team has to cancel their own proposal. This means that any proposal that cancels another proposal will be processed before all others. Anyone can submit an AGP. This means that every quarter every team (current or new) will be incentivized to submit an AGP cancelling the funding of other teams. They can do this transparently or anonymously. 
```

Comment:
The fact that an AGP has been reversed yet isn't really a good argument for anything. The AGP process is barely a year old, it's not unusual that authors and voters haven't found an AGP worthy of completely undoing yet. However, AGPs have been approved that _modify_ previous AGPs (all the approved Meta track AGPs for example), showing that voters are willing to introspect and make changes when needed.

Reply:
Good call. Updated accordingly.